internal/utils/utils.go

```diff
@@ -7,6 +7,8 @@ import (
 	resourceschema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
 )
 
+const limit int32 = 1000
```

Comment:
can you please rename to `ReadPageSizeLimit` ?

Reply:
fixed